examples/ex1p.cpp

```diff
@@ -84,6 +85,8 @@ int main(int argc, char *argv[])
                   "--no-static-condensation", "Enable static condensation.");
    args.AddOption(&pa, "-pa", "--partial-assembly", "-no-pa",
                   "--no-partial-assembly", "Enable Partial Assembly.");
+   args.AddOption(&amgx, "-amgx", "--amgx-precon", "-no-amgx",
+                  "--no-amgx-precon", "Use AmgX.");
```

Comment:
A slightly longer help line would be nice, maybe "Use AmgX V-cycle as preconditioner for CG."